src/components/pages/FoundationFooter.astro

```diff
@@ -61,7 +61,7 @@ const currentYear = new Date().getFullYear();
   <ul class="footer-links">
     <li><a href="/">Terms & conditions</a></li>
     <li><a href="https://www.iubenda.com/privacy-policy/88624819">Privacy policy</a></li>
-    <li><a href="">Contact us</a></li>
+    <li><a href="/contact">Contact us</a></li>
```

Comment:
Fix missing link.
Will have to chase Anna about the terms & conditions link, this is an issue for the main site as well.